package.json

```diff
@@ -44,6 +44,7 @@
   },
   "dependencies": {
     "babel-runtime": "^5.6.20",
+    "graphql-relay": "^0.3.2",
```

Comment:
So are you suggesting we move it to dev dependencies and if the user tries to require from the relay file they will get an error? I don't think this is the best case and graphql-relay is a requirement for the work we've done thus far.